### DISCUSSION TITLE
Dropdown Menu Trigger automatically highlights when blurring the menu

### DISCUSSION THREAD

**Discussion by ssebastianoo** [shadcn-ui/ui#3313] [Q&A] [answered]
Why does the dropdown menu trigger highlights everytime I close the dropdown?

https://github.com/shadcn-ui/ui/assets/48064277/0990b62b-5814-4ddf-bccc-0a0774c16786

**Comment by jovanblazek** [comment, marked as answer]
Try adding `onCloseAutoFocus` on `DropdownMenuContent`. It solved it for me
```tsx
<DropdownMenuContent onCloseAutoFocus={(e) => e.preventDefault()}>
```

**Reply by K0IIIMAP** [reply]
thanks sir

**Comment by ice7mayu** [comment]
And this issue is still not fixed

**Reply by diecodev** [reply]
this is a discussion, no issue.
